src/compliance.rs

```diff
@@ -0,0 +1,293 @@
+use colored::Colorize;
+use serde::Deserialize;
+
+use crate::{make_github_request, Bootstrap};
+
+#[derive(Default, Debug, Clone, Copy)]
+struct ProtectionChecks {
+    pr_one_approval: bool,
+    pr_dismiss_stale: bool,
+    pr_require_code_owner: bool,
+    disable_force_push: bool,
+    disable_deletion: bool,
+    require_signed_commits: bool,
+    require_status_checks: bool,
+}
+
+impl ProtectionChecks {
+    fn score(&self) -> u8 {
```

Comment:
Why is this a `u8`? This will become a problem in the future if you ever want things to be weighted differently.

Reply:
@obelisk good callout - updated to u32